keyboards/yoichiro/lunakey_pico/info.json

```
@@ -0,0 +1,97 @@
+{
+    "keyboard_name": "Lunakey Pico",
+    "manufacturer": "yoichiro",
+    "maintainer": "Yoichiro Tanaka",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": false,
+        "rgblight": true,
+        "audio": false
+    },
+    "matrix_pins": {
+        "cols": ["GP21", "GP20", "GP19", "GP18", "GP17", "GP16"],
+        "rows": ["GP12", "GP13", "GP14", "GP15"]
+    },
+    "debounce": 5,
+    "processor": "RP2040",
+    "usb": {
+        "device_version": "3.0.0",
+        "pid": "0x0003",
+        "vid": "0x5954"
+    },
+    "rgblight": {
+        "led_count": 12,
+        "pin": "GP6",
+        "sleep": true,
+        "split": true,
+        "split_count": [6, 6],
+        "animations": {
+            "all": true
```

Could you explicitly define the animations:
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": false,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```